mteb/tasks/Classification/multilingual/CyrillicTurkicLangClassification.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class CyrillicTurkicLangClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CyrillicTurkicLangClassification",
+        description="Cyrillic dataset of 8 Turkic languages spoken in Russia and former USSR",
+        dataset={
+            "path": "tatiana-merz/cyrillic_turkic_langs",
+            "revision": "e42d330f33d65b7b72dfd408883daf1661f06f18",
+        },
+        reference="https://huggingface.co/datasets/tatiana-merz/cyrillic_turkic_langs",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=[
+            "bak-Cyrl",  # Bashkir
+            "chv-Cyrl",  # Chuvash
+            "tat-Cyrl",  # Tatar
+            "kir-Cyrl",  # Kyrgyz
+            "rus-Cyrl",  # Russian
+            "kaz-Cyrl",  # Kazakh
+            "tyv-Cyrl",  # Tuvinian
+            "krc-Cyrl",  # Karachay-Balkar
+            "sah-Cyrl",  # Yakut
+        ],
+        main_score="accuracy",
+        date=("2012-01-01", "2012-05-01"),
```

Comment:
This should be the time that the text were written. My guess is that it expands further back.